tools.yml

```diff
@@ -1190,7 +1190,7 @@ tools:
     mem: 40
   toolshed.g2.bx.psu.edu/repos/iuc/funannotate_clean/funannotate_clean/.*:
     cores: 8
-    mem: 40
+    mem: 64
```

Comment:
@TomHarrop do you know how large? can we make it conditional based on the input size?

Reply:
this was the wrong line in the file, sorry :facepalm: